src/Entity/Usuario.php

```diff
@@ -110,6 +113,17 @@ public function setPassword(string $password): static
         return $this;
     }
 
+    public function taAtivo(): bool
```

Comment:
geralmente usamos assim
```suggestion
    public function isAtivo(): bool
```

Reply:
na verdade nem precisa, já tem um método para isso, remove ele e usa o outro!

Reply:
Estava usando assim, mas por algum motivo, quando coloquei "isAtivo", ele simplesmente dava erro, daí foi o jeito eu deixar dessa forma.

Reply:
Ele deu erro porque o método já existia na classe